com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/ZigBeeNode.java

```diff
@@ -420,7 +420,7 @@ public ZigBeeStatus call() throws Exception {
         });
 
         // start the thread to execute it
-        new Thread(future).start();
+        new Thread(future, "UpdateBindingTable" + getIeeeAddress()).start();
```

Comment:
Isn't here a separator missing between the "UpdateBindingTable" and the address?

Reply:
Actually, this was intended - I figured it was obvious enough without the separator, but you're right it is neater so I will update.

Thanks for the review.